tests/InstrumentationEngine.Lib.Tests/InstrumentationEngine.Lib.Tests.vcxproj

```diff
@@ -91,7 +91,6 @@
     <ClInclude Include="TestLoggingHost.h" />
   </ItemGroup>
   <ItemGroup>
-    <ClCompile Include="ConfigurationLoaderTests.cpp" />
```

Comment:
>ConfigurationLoaderTests [](http://example.com/codeflow?start=24&length=24)

was this removal intentional?

Reply:
Yes. It was moved to the cross plat tests.

Reply:
Looks like I forgot to delete the cpp file from this directory though.